packages/main/test/specs/TimeSelectionClocks.spec.js

```diff
@@ -0,0 +1,360 @@
+import { assert } from "chai";
+
+describe("Interactions", () => {
+	before(async () => {
+		await browser.url(`test/pages/TimeSelectionClocks.html`);
```

Comment:
Could you try opening the "TimeSelectionClocks.html" just once in the Interactions section, and not opening the page again in Accessibility and Events. If you need some kind of resetting, you can use other instances if the time selection clock. It just that we noticed reopening of the page slows down test execution and the PR checks.